pyinstaller/specter_desktop.spec

```diff
@@ -15,6 +15,9 @@ elif platform.system() == 'Linux':
         binaries = [("/lib/aarch64-linux-gnu/libusb-1.0.so.0", ".")]
     else:
         binaries = [("/lib/x86_64-linux-gnu/libusb-1.0.so.0", ".")]
+    binaries.extend([("/usr/lib/x86_64-linux-gnu/dri/iris_dri.so","."),\
+                     ("/usr/lib/x86_64-linux-gnu/gio/modules/libgvfsdbus.so","."),\
+                     ("/usr/lib/x86_64-linux-gnu/gvfs/libgvfscommon.so",".")])
```

Comment:
This will fail on aarch64 (arm). Even though we are not building desktop app on ARM yet we may do it in the future.
Let's use the path based on the processor architecture:
```py
    arch = platform.processor()
    binaries = [(f"/lib/{arch}-linux-gnu/libusb-1.0.so.0", "."),
                (f"/usr/lib/{arch}-linux-gnu/dri/iris_dri.so","."),
                (f"/usr/lib/{arch}-linux-gnu/gio/modules/libgvfsdbus.so","."),
                (f"/usr/lib/{arch}-linux-gnu/gvfs/libgvfscommon.so","."),
    ]
```